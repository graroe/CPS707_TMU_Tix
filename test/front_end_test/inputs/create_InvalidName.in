Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Enter date of event <format YYYYMMDD>: 
Enter number of available tickets <max 9999>: 
Event CirqueDuSoliel created.
Enter new event name <max 15 chars>: 
Event name already exists
Enter new event name <max 15 chars>: 
That command is not a valid event name. Enter !q to cancel new event creation.
Enter new event name <max 15 chars>: 
Character limit exceeded
Enter new event name <max 15 chars>:
Transaction canceled.
Session terminated.