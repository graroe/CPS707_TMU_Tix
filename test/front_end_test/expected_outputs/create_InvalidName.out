Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Event name already exists\nEnter new event name <max 15 chars>: 
Character limit exceeded\nEnter new event name <max 15 chars>: 
Transaction canceled
Session terminated.